### PR TITLE
Move Tezos to the latest-release branch

### PR DESCRIPTION
**Pull request by emmanueldenloye**
Also remove the botttle. I'm not intimately familiar with ruby, but
that seems to work and generate the binaries.

### CODE REVIEW

**Review by michaeljklein — APPROVED** [review]
LGTM

**Comment by michaeljklein** [comment]
@itkach The version can be made more descriptive, but it'll require new bottles.

File: Formula/tezos.rb [hunk 1]
```diff
@@ -8,16 +8,17 @@ class << self
   desc "Platform for distributed consensus with meta-consensus capability"
   homepage "https://gitlab.com/tezos/tezos"
 
-  url "https://gitlab.com/tezos/tezos.git", :revision => "64c855499057fa84b41fcb9f3d7d72d4570db1a7", :branch => "mainnet", :shallow => false
-  version "006_PsCARTHA"
+  url "https://gitlab.com/tezos/tezos.git", :revision => "51977265590ba5fbd166b921e265fa22bf9f66a6", :branch => "latest-release", :shallow => false
 
-  head "https://gitlab.com/tezos/tezos.git", :branch => "carthagenet"
+  version "007_multinetwork"
```
Comment (itkach) [inline Formula/tezos.rb]: I think it's important that version reflects tezos version as provided by upstream, currently 7.2

File: Formula/tezos.rb [hunk 2]
```diff
@@ -8,9 +8,9 @@ class << self
   desc "Platform for distributed consensus with meta-consensus capability"
   homepage "https://gitlab.com/tezos/tezos"
 
-  url "https://gitlab.com/tezos/tezos.git", :revision => "51977265590ba5fbd166b921e265fa22bf9f66a6", :branch => "latest-release", :shallow => false
+  url "https://gitlab.com/tezos/tezos.git", :revision => "6b9f3bc3c90d99d4a2bd5cd26e6f3efbcb977090", :branch => "latest-release", :shallow => false
 
-  version "007_multinetwork"
+  version "7.2-release"
```
Comment (itkach) [inline Formula/tezos.rb]: version is just "7.2", not 7.2-release

File: Formula/tezos.rb [hunk 3]
```diff
@@ -8,16 +8,17 @@ class << self
   desc "Platform for distributed consensus with meta-consensus capability"
   homepage "https://gitlab.com/tezos/tezos"
 
-  url "https://gitlab.com/tezos/tezos.git", :revision => "64c855499057fa84b41fcb9f3d7d72d4570db1a7", :branch => "mainnet", :shallow => false
-  version "006_PsCARTHA"
+  url "https://gitlab.com/tezos/tezos.git", :revision => "6b9f3bc3c90d99d4a2bd5cd26e6f3efbcb977090", :branch => "latest-release", :shallow => false
```
Comment (itkach) [inline Formula/tezos.rb]: is it necessary to specify branch if we have revision?
also, this revision corresponds to tag `v7.2`, can we use tag instead?

**Review by itkach — APPROVED** [review]

**Pull request merged**